compose/config/config.py

```diff
@@ -76,6 +76,7 @@
     'fig.yaml',
 ]
 
+DEFAULT_OVERRIDE_FILENAME = 'docker-compose.override.yml'
```

Comment:
I think this belongs with the list of `SUPPORTED_FILENAMES` which is just above this in this module.

`const.py` is good for constants that are used in multiple places, but if a value is only used in a single module (like this one) I prefer to keep it together with the code that uses it.